File: 4기-코딩테스트-220608/4기 코딩테스트_220608/4기 - 1st 코딩테스트/CountPrefixOddWordNumber/CountPrefixOddWordNumber.py
def solution(array, s):
    """
    :param array: string[]
    :param s: string
    :return: int
    """

    result = 0

    for prefix in array:
        if len(prefix) % 2 == 0:
            continue
        if s.find(prefix) == 0:
            result += 1

    return result

File: 4기 코딩테스트_220615/4기 - 1st 코딩테스트/CountPrefixOddWordNumber/CountPrefixOddWordNumber.js
/**
 * @param array {string[]}
 * @param s {string}
 * @returns {number}
 */
function solution(array, s) {
  let result = 0

  for (let i = 0; i < array.length; i++) {
    const prefix = array[i]
    if (prefix.length % 2 === 0) continue
    if (s.indexOf(prefix) === 0) {
      result++
    }
  }

  return result
}

export default solution
